config/default.yml

```
@@ -199,6 +199,16 @@ Rails/ContentTag:
   Exclude:
     - app/models/**/*.rb
 
+Rails/ControllerTesting:
+  Description: 'Use `ActionDispatch::IntegrationTest` instead of `ActionController::TestCase`.'
```

Can you add the style guide reference?

```suggestion
  Description: 'Use `ActionDispatch::IntegrationTest` instead of `ActionController::TestCase`.'
  StyleGuide: 'https://rails.rubystyle.guide/#integration-testing'
```